### ISSUE TITLE
Making a new feature for a spawn area in single world plotworld's

### ISSUE THREAD

**Issue by LPGamersRob** [WorldCretornica/PlotMe-Core#217]
hi there.

i know this seems like a lot of work but i actually love plotme but i'm having a stress time a the moment with making a singular world spawn area, I desperately want to keep away from making a multiverse server where i have a separate world for my spawn.

i was thinking is there any way that a feature could be added where as you can use worldedit on the plotworld to select an area and do a command like (/plotme spawn) and this removes any of the plotme features in that area like being able to claim the plots in the (/p spawn) area so that people could put third party plugins on there like parkour and npc without having the trouble of plotme over powering all the plugins features.

if this is not clear let me know and ill try abit better to explain what i mean!

**Comment by MattBDev** [comment]
If you can elaborate a bit and if needed, use pictures to explain, it would help me a bit :)

**Comment by LPGamersRob** [comment]
okay so if i was in the plot world, at the moment i would have to make a spawn out of the plots that are there, but this is restrictive because of the world guard that plotme uses to make plots unbreakable.
so using plots is a hard thing to do as you have to allow players access to the plots so that they can use third party plugins, but this also give them access to break the spawn then,

so my idea is that there is a command that can make an area that is like having an anti-plot zone that is not affected by the plotme behaviours.

**Comment by MattBDev** [comment]
If I understand correctly, you want an ability to merge plots to create a spawnpoint built of multiple plots because the restriction of not building on roads makes it difficult to make a nice spawnpoint?

**Comment by LPGamersRob** [comment]
yes that is essentially what i mean yes but with the ability of that combined plot to have the world guard removed from it.

**Comment by jomo** [comment]
related: #186

**Comment by EvilOlaf** [comment]
So it seems to be all about plot merging. That is a quite long expected feature and will eventually integrated....well, I hope so ;)

The opposite feature might be to be able to disable the plotme protection for an area at all so in this area everyone could build without any restrictions. In these areas further controls about user activity could be enforced by using a 3rd party plugin like WorldGuard.
This might not match the project idea to not need any additional plugins...well it would be a soft dependency ;)

**Comment by MattBDev** [comment]
This idea is similar to plot merging but it is possible for it to be accomplished without plot merging actually working. I want to get more feedback on this before I do work on this.

**Comment by LPGamersRob** [comment]
i think that ill make a small video for this on what i mean because i think i can talk about this easyer than i can type ? 

Thank you all for feedback and help though

**Comment by LPGamersRob** [comment]
hi again i have now made the video.. it is on youtube Here is the link 

https://www.youtube.com/watch?v=vxozIdYZ2-o&feature=youtu.be

**Comment by MattBDev** [comment]
Good video. I understand what you are saying and I will put this into PlotMe-Core as soon as it is stable enough to handle it, meaning after 0.17 but I will add this or something extremely similar.

**Comment by EvilOlaf** [comment]
Nice.
It seems that this is basically that what I described above as the opposite feature :D

**Comment by LPGamersRob** [comment]
yes it is what you described ... and thank you for all your help and i do look forward to this being added it would make life alot easyer :)
